arviz/plots/kdeplot.py

```diff
@@ -4,9 +4,9 @@
 import matplotlib.pyplot as plt
 from scipy.signal import gaussian, convolve, convolve2d  # pylint: disable=no-name-in-module
 from scipy.sparse import coo_matrix
+from numba import njit
```

Comment:
do we still keep numba as an optional dependency?

Reply:
I think we want to keep this optional. not sure how easy is to install numba if not using conda

Reply:
I think it is not too bad, but arch and other smaller distros could have a problem